bats/tests/compose/testdata/compose.yaml

```
@@ -0,0 +1,9 @@
+services:
+  web: 
+    build:
+      context: app
+    # flask requires SIGINT to stop gracefully
+    # (default stop signal from Compose is SIGTERM)
+    stop_signal: SIGINT
+    ports:
+      - '8000:8000'
```

Stylistic preference: don't do the double-indentation of list elements. It is valid, but looks weird.

```suggestion
    ports:
    - '8000:8000'
```